pkg/registry/resolver_test.go

```diff
@@ -711,6 +713,202 @@ func TestResolve(t *testing.T) {
 				},
 			}},
 		},
+	}, {
+		name: "Workflow with Test and ClusterClaim overridden",
+		config: api.MultiStageTestConfiguration{
+			Workflow: &awsWorkflow,
+			ClusterClaim: &api.ClusterClaim{
+				Architecture: "arm64",
+				Cloud:        "gcp",
+				Owner:        "openshift-ci2",
+				Product:      "rhcos",
+				Timeout:      &prowv1.Duration{Duration: time.Minute},
+				Version:      "4.11",
+			},
+			Test: []api.TestStep{{
+				LiteralTestStep: &api.LiteralTestStep{
+					As:       "custom-e2e",
+					From:     "test-image",
+					Commands: "make custom-e2e-2",
+					Resources: api.ResourceRequirements{
+						Requests: api.ResourceList{"cpu": "1000m"},
+						Limits:   api.ResourceList{"memory": "2Gi"},
+					}},
+			}},
+		},
+		workflowMap: WorkflowByName{
+			awsWorkflow: {
+				ClusterClaim: &api.ClusterClaim{
+					Architecture: "amd64",
+					Cloud:        "aws",
+					Owner:        "openshift-ci",
+					Product:      "ocp",
+					Timeout:      &prowv1.Duration{Duration: time.Hour},
+					Version:      "4.10",
+				},
+				Pre: []api.TestStep{{
+					LiteralTestStep: &api.LiteralTestStep{
+						As:       "ipi-install",
+						From:     "installer",
+						Commands: "openshift-cluster install",
+						Resources: api.ResourceRequirements{
+							Requests: api.ResourceList{"cpu": "1000m"},
+							Limits:   api.ResourceList{"memory": "2Gi"},
+						}},
+				}},
+				Test: []api.TestStep{{
+					LiteralTestStep: &api.LiteralTestStep{
+						As:       "e2e",
+						From:     "my-image",
+						Commands: "make custom-e2e",
+						Resources: api.ResourceRequirements{
+							Requests: api.ResourceList{"cpu": "1000m"},
+							Limits:   api.ResourceList{"memory": "2Gi"},
+						}},
+				}},
+				Post: []api.TestStep{{
+					LiteralTestStep: &api.LiteralTestStep{
+						As:       "ipi-teardown",
+						From:     "installer",
+						Commands: "openshift-cluster destroy",
+						Resources: api.ResourceRequirements{
+							Requests: api.ResourceList{"cpu": "1000m"},
+							Limits:   api.ResourceList{"memory": "2Gi"},
+						}},
+				}},
```

Comment:
These are just noise in the definitions of the input/output, they can be removed in all test cases.